src/Chapter1.hs

```diff
@@ -620,7 +636,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = firstLast + secondLast
+  where
+    a = abs n
+    firstLast = mod a 10
+    secondLast = mod (div a 10) 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (mod x 100) 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Woow, I don't know about this pattern matching way 0_o
```
(first, second) = divMod (mod x 100) 10
```

Reply:
Don't worry! We explain pattern matching in Chapter Two :slightly_smiling_face: